cluster/manifests/node-problem-detector/ds-node-problem-detector.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: node-problem-detector
+  namespace: kube-system
+  labels:
+    application: node-problem-detector
+    version: v0.6.3
+spec:
+  selector:
+    matchLabels:
+      application: node-problem-detector
+      version: v0.6.3
+  template:
+    metadata:
+      labels:
+        application: node-problem-detector
+        version: v0.6.3
+    spec:
+      containers:
+      - name: node-problem-detector
+        image: k8s.gcr.io/node-problem-detector:v0.6.3
+        command:
+        - "/bin/sh"
+        - "-c"
+        # Pass both config to support both journald and syslog.
+        - "exec /node-problem-detector --logtostderr --system-log-monitors=/config/kernel-monitor.json,/config/kernel-monitor-filelog.json,/config/docker-monitor.json,/config/docker-monitor-filelog.json"
+        securityContext:
+          privileged: true
+        resources:
+          limits:
+            cpu: "200m"
+            memory: "100Mi"
+          requests:
+            cpu: "20m"
+            memory: "20Mi"
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        volumeMounts:
+        - name: log
+          mountPath: /var/log
```

Comment:
Shouldn't this be readonly?

Reply:
The old config used to write logs to a file :open_mouth: in `/var/log` itself. Changed it.